src/plugins/intel_npu/src/compiler_adapter/include/vcl_api.hpp

```diff
@@ -0,0 +1,130 @@
+// Copyright (C) 2018-2025 Intel Corporation
```

Comment:
```suggestion
// Copyright (C) 2025 Intel Corporation
```
I think this is the right format since the file new.

Reply:
update in new commit [32fa2f8](https://github.com/openvinotoolkit/openvino/pull/30732/commits/32fa2f870d1eb5de8569123e702beb692693f8f6)